src/app/rpmostree-builtin-livefs.cxx

```diff
@@ -47,7 +47,7 @@ get_args_variant (GError **error)
   if (opt_target)
     {
       if (opt_reset)
-        return glnx_null_throw (error, "Cannot specify both --target and --reset");
+        return glnx_null_throw (error, "Cannot specify both --target and --reset"), nullptr;
```

Comment:
Hmm, I'd personally prefer just casting it like we do elsewhere.

Reply:
OK, done.  The advantage of `nullptr` is that it auto-casts.  Hm, I guess I could just fix libglnx upstream to detect C++ and return `nullptr`.

Reply:
But really I was trying to minimize churn; the idea is this code gets ported to Rust (or perhaps C++ exceptions).